modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-options/src/main/resources/META-INF/resources/options_field.js

```diff
@@ -341,6 +342,14 @@ AUI.add(
 						mainOption.set('errorMessage', event.newVal);
 					},
 
+					_afterOptionKeyChange: function(event) {
```

Comment:
Hey @pedroqueiroz could you remove the event argument? As result of it isn't using in this method